app/models/reference_payment_updater.rb

```diff
@@ -41,8 +41,8 @@ def save_config_reference_number
   def save_config_payment_link
     if reference_payment_settings.payment_link_url_present? &&
         reference_payment_settings.payment_link_checked?
-      create_or_update_service_configuration(config: 'PAYMENT_LINK', deployment_environment: 'dev', value: payment_link_url)
-      create_or_update_service_configuration(config: 'PAYMENT_LINK', deployment_environment: 'production', value: payment_link_url)
+      create_or_update_service_configuration(config: 'PAYMENT_LINK', deployment_environment: 'dev', value: payment_link_url.delete(' '))
+      create_or_update_service_configuration(config: 'PAYMENT_LINK', deployment_environment: 'production', value: payment_link_url.delete(' '))
```

Comment:
use `strip` and this is probably best done in the `reference_payment_settings` model in the `payment_link_url` method